app/views/roles/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<% content_for(:title, "Roles") %>
+
+<% if @roles.empty? %>
+  <div class="none-found">
+    No roles found.
+  </div>
+<% else %>
+  <% @roles.each do |role| %>
+    <h3><%= role.name.humanize %></h3>
+    <div id="directory-album" class="flowing-columns-max-4 hidden-print">
```

Comment:
so this is all copied from the people view. Should we extract it somewhere common, or is that ok?

Reply:
I don't think you need the directory-album ID, do you?

Reply:
Also don't need hidden-print.